chart/otomi/templates/job.yaml

```diff
@@ -20,8 +20,8 @@ spec:
       initContainers:
         - name: bootstrap-values
           securityContext: {{- toYaml .Values.securityContext | nindent 12 }}
-          image: otomi/core:{{ .Values.otomi.version }}
-          imagePullPolicy: Always
+          image: "{{ .Values.coreImage.repository | default "otomi/core" }}:{{ .Values.coreImage.tag | default .Values.otomi.version }}"
```

Comment:
It is a bit inconsistent that you specify default for `image` here and default for `imagePullPolicy` in values.yaml.
Could we follow one pattern?
Now I am also wondering if the `otomi` object should contain imagePullPolicy, repository and version. What do you think?

Reply:
just follow the helm chart paradigm

Reply:
if that exists, but being pragmatical I mean to look at `helm create xx` and inspect `xx`